separate daily and main workflows

notmandatory opened the pull request.

notmandatory commented:
BTW, I left the instructions for installing to still compile from code so users will always get the latest updates, since I suspect features will still be evolving a bit.

afilini commented:
I'd say everything you've done looks good, the only missing thing that I can think of is the shortcut for the playground in the menu in `config.toml`:

https://github.com/bitcoindevkit/bitcoindevkit.org/blob/a495843b17406b63c3a0c63900fe4e88b7b80ec0/config.toml#L59-L62

notmandatory commented:
> I'd say everything you've done looks good, the only missing thing that I can think of is the shortcut for the playground in the menu in `config.toml`:
> 
> https://github.com/bitcoindevkit/bitcoindevkit.org/blob/a495843b17406b63c3a0c63900fe4e88b7b80ec0/config.toml#L59-L62

Oh ya! I should have tested that, fixed and I moved the  `static/repl` stuff to `static/bdk-cli` also. Verified playground links are both working.

afilini commented:
Looks good, ACK d2518b98217afbbfd9c77a2c40f690f8c72887a6

notmandatory commented:
Sorry had to rebase on latest `master` to pickup nightly autogen'd doc updates
